inc/scroom/viewinterface.hh

```diff
@@ -234,16 +228,12 @@ public:
   /**
    * Enable panning the view.
    */
-  virtual void setPanning()
-  {
-  }
+  virtual void setPanning()=0;
 
   /**
    * Disable panning the view.
    */
-  virtual void unsetPanning()
-  {
-  }
+  virtual void unsetPanning()=0;
```

Comment:
Now that selections are done with the right mouse button, are these still necessary?

Reply:
I don't think there's a use case left for them and judging by your comment I guess you'd prefer them gone so I'll remove them later. If for some unlikely reason we end up needing them again then adding them back isn't a lot of effort either.